.github/workflows/ci.yml

```diff
@@ -13,55 +13,33 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  ci:
-    name: CI
+  typecheck:
+    name: Typecheck
+    uses: ./.github/workflows/typecheck.yml
+
+  lint:
+    name: Lint 
+    uses: ./.github/workflows/lint.yml
```

Comment:
I dont think you'd need this? coz all the workflows are already inside the `workflow` dir right?